client/login/magic-login/handle-emailed-link-form-jetpack-connect.tsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * External dependencies
+ */
+import React, { FC, useCallback, useEffect, useState } from 'react';
+import classNames from 'classnames';
```

Comment:
```suggestion
```

'classNames' is defined but never used.  It's causing a TeamCity Code Style failure.

Reply:
Good catch, Bryan!